docs/howtos/releases.md

```diff
@@ -145,8 +145,9 @@ If you want to uplift changes into a previous release:
 
 * Make sure the changes are present in `main` and have been thoroughly tested
 * Checkout the `release-vXXX` branch, where `XXX` is the major version number
-* Create a PR to bump the version in `version.txt` from [release_version].0 to [release_version].0.1 on the release branch
+  * Do not bump `version.txt`, this is handled by mergify.
```

Comment:
It's handled by Release Management. That said, I don't know why it can't be bumped automatically as part of the Ship-It graph like we do for other projects.